config/scout.php

```diff
@@ -139,4 +139,38 @@
         ],
     ],
 
+    /*
+    |--------------------------------------------------------------------------
+    | Typesense Configuration
+    |--------------------------------------------------------------------------
+    |
+    | Here you may configure your Typesense settings. Typesense is an open
+    | source search engine with minimal configuration. Below, you can state
+    | the host and key information for your own Typesense installation.
+    |
+    | See: https://typesense.org/docs/0.25.1/api/authentication.html
+    |
+    */
+    'typesense' => [
+        'api_key'         => env('TYPESENSE_API_KEY', 'xyz'),
```

Comment:
Please only use one space before `=>` assignments.

```suggestion
        'api_key' => env('TYPESENSE_API_KEY', 'xyz'),
```